lib/model/katex.dart

```diff
@@ -688,6 +761,196 @@ class KatexSpanStyles {
   }
 }
 
+final _hexColorRegExp =
+  RegExp(r'^#([0-9a-fA-F]{3}|[0-9a-fA-F]{4}|[0-9a-fA-F]{6}|[0-9a-fA-F]{8})$');
+
+/// Parses the CSS hex color notation.
+///
+/// See: https://drafts.csswg.org/css-color/#hex-notation
+KatexSpanColor? parseCssHexColor(String hexStr) {
+  final match = _hexColorRegExp.firstMatch(hexStr);
+  if (match == null) return null;
+
+  String hexValue = match.group(1)!;
+  hexValue = hexValue.toLowerCase();
```

Comment:
Is this line needed? I'd have thought that `hex.decode` would accept upper as well as lower case.